rmw_connext_dynamic_cpp/src/functions.cpp

```diff
@@ -198,9 +198,20 @@ rmw_init()
 }
 
 rmw_node_t *
+rmw_create_secure_node(
+  const char * name,
+  const char * namespace_,
+  size_t domain_id,
+  const char * security_files_paths)
+{
+  (void)security_files_paths;
+  return create_node(rti_connext_dynamic_identifier, name, namespace_, domain_id, NULL);
```

Comment:
given that connext_dynamic is broken I don't know if this works or not so I didn't pass the variable. I'll pass it anyway given that the code path for connext and connext dynamic is exactly the same in the case of node creation. Thanks for pointing it out